sympy/physics/control/lti.py

```diff
@@ -1843,19 +1905,398 @@ def doit(self, **kwargs):
+        [------------------------  --------------------------]
+        [   4      3                   5      4      2       ]
+        [  p  - 3*p  + 2*p - 1        p  - 3*p  + 2*p  - p   ]
+        [                                                    ]
+        [     4    3    2               4      3             ]
+        [    p  - p  - p  + p        2*p  - 3*p  + 2*p - 1   ]
+        [  -------------------       ---------------------   ]
+        [   4      3                   4      3              ]
+        [  p  - 3*p  + 2*p - 1        p  - 3*p  + 2*p - 1    ]
+
+        """
+        _plant_mat = self.plant.doit()._expr_mat
+        _controller_mat = self.feedback_controller.doit()._expr_mat
+
+        return (eye(self.plant.num_inputs) - \
+            self.ftype*_plant_mat*_controller_mat).inv()
+
+    def doit(self, cancel=True, expand=False, **kwargs):
+        r"""
+        Returns the resultant closed loop transfer function matrix obtained by the
+        closed-loop feedback interconnection.
+
+        Examples
+        ========
+
+        >>> from sympy import pprint
+        >>> from sympy.abc import s
+        >>> from sympy.physics.control.lti import TransferFunction, TransferFunctionMatrix, MIMOFeedback
+        >>> tf1 = TransferFunction(s, 1 - s, s)
+        >>> tf2 = TransferFunction(1, s, s)
+        >>> tf3 = TransferFunction(10, 1, s)
+        >>> tf4 = TransferFunction(s - 1, 1, s)
+        >>> tf5 = TransferFunction(0, 1, s)
+        >>> sys1 = TransferFunctionMatrix([[tf1, tf2], [tf3, tf4]])
+        >>> sys2 = TransferFunctionMatrix([[tf3, tf5], [tf5, -tf5]])
+        >>> F_1 = MIMOFeedback(sys1, sys2)
+        >>> pprint(F_1, use_unicode=False)
+        /    [  s      1  ]    [10  0]   \-1 [  s      1  ]
+        |    [-----    -  ]    [--  -]   |   [-----    -  ]
+        |    [1 - s    s  ]    [1   1]   |   [1 - s    s  ]
+        |I + [            ]   *[     ]   |  *[            ]
+        |    [ 10    s - 1]    [0   0]   |   [ 10    s - 1]
+        |    [ --    -----]    [-   -]   |   [ --    -----]
+        \    [ 1       1  ]{t} [1   1]{t}/   [ 1       1  ]{t}
+        >>> pprint(F_1.doit(), use_unicode=False)
+        [                 s                                  1 - s              ]
+        [              -------                            -----------           ]
+        [              9*s + 1                            s*(9*s + 1)           ]
+        [                                                                       ]
+        [100*s*(s - 1) + 10*(1 - s)*(9*s + 1)  s*(s - 1)*(9*s + 1) + 100*s - 100]
+        [------------------------------------  ---------------------------------]
+        [         (1 - s)*(9*s + 1)                       s*(9*s + 1)           ]{t}
+
+        If the user wants the the resultant ``TransferFunctionMatrix`` object without
+        cancelling the common factors then the ``cancel`` arg should be passed ``False``.
```

Comment:
```suggestion
        If the user wants the resultant ``TransferFunctionMatrix`` object without
        canceling the common factors then the ``cancel`` arg should be passed ``False``.
```
Can we provide an example where it actually avoids canceling the common factors? It is not apparent in this one.

Reply:
Done